documentation/configuration.markdown

```diff
@@ -35,6 +35,8 @@ The default Logback configuration file is at `/etc/puppetserver/logback.xml` or
 
 Puppet Server relies on `logrotate` to manage the log file, and installs a configuration file at `/etc/logrotate.d/puppetserver` or `/etc/logrotate.d/pe-puppetserver`.
 
+For some tips on advanced logging configuration, including information about configuring your system to send log data to logstash, see the [Advanced Logging Configuration](./config_logging_advanced.html) documentation.
```

Comment:
I agree that we could use more coverage of other logback features, and would love to see your branch get merged in wherever is appropriate.  One minor thing to keep in mind is that much of this documentation is relevant to all of the other Clojure projects as well, but we could at least start by getting it into the Puppet Server docs.  Your branch looks great, let us know how we can help.

I will tweak the mention of logstash above to be a bit more general.